hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-router/src/main/java/org/apache/hadoop/yarn/server/router/clientrm/RouterClientRMService.java

```diff
@@ -157,6 +163,11 @@ protected void serviceStart() throws Exception {
             YarnConfiguration.DEFAULT_ROUTER_CLIENTRM_ADDRESS,
             YarnConfiguration.DEFAULT_ROUTER_CLIENTRM_PORT);
 
+    if (getConfig().getBoolean(YarnConfiguration.ROUTER_WEBAPP_PROXY_ENABLE,
```

Comment:
Let's use `conf`, We have defined this variable.

```
conf.getBoolean(YarnConfiguration.ROUTER_WEBAPP_PROXY_ENABLE
```

Reply:
I will fix it.